can/io/log.py

```diff
@@ -0,0 +1,95 @@
+from can.listener import Listener
+import datetime
+import time
+from can.message import Message
+
+CAN_MSG_EXT = 0x80000000
+CAN_ERR_FLAG = 0x20000000
+CAN_ERR_BUSERROR = 0x00000080
+CAN_ERR_DLC = 8
+
+class canutilsLogReader(object):
+    """
+    Iterator of CAN messages from a .log Logging File (candump -L).
+
+    Only CAN messages are supported (no Errorframes).
+    """
+
+    def __init__(self, filename):
+        self.fp = open(filename, "r")
+
+    def __iter__(self):
+        for line in self.fp:
+            temp = line.strip()
+            if len(temp) > 0:
+                (timestamp, bus, frame) = temp.split()
+                timestamp = float(timestamp[1:-1])
```

Comment:
You want to skip the first and last character of `timestamp` before parsing as a float - Is this what you want? What are those characters? Can we `strip()` to make this clearer? Otherwise I think it deserves a comment for people like me who are unfamiliar with the format.

Reply:
maybe somewhere a example .log-Entry should be documented:
.log-file looks like:
(0.0) vcan0 001#8d00100100820100
after splitting (0.0) remains, thus I need [1:-1] to cut of the braces

Reply:
Ok that sounds reasonable, I think it would be slightly clearer to use `strip("()")`